foqus_lib/framework/sdoe/df_utils.py

```diff
@@ -8,7 +9,7 @@ def write(fname, df):
 
 def load(fname):
     # load file as data frame
-    df = pd.read_csv(fname)
+    df = pd.read_csv(os.path.join('c:/', fname))
```

Comment:
This looks OS-specific.  What will happen here on a non-Windows platform?

Reply:
That was my first thought, but I tested it on Mac and still works...

Reply:
I verified that this fixed the SDOE related aspect of Issue #396